### PR TITLE
CONN-2013-Apply HSTS Patch into 4.5.3

### PR DESCRIPTION
**Pull request by mhpnguyen**
Only apply to PD,DQ,DR services
@sovanncgi (optional)

### CODE REVIEW

**Review by TabassumJafri — COMMENTED** [review]
Reviewed PR .. will approve once the forgerock issue is fixed for the build.

**Comment by mhpnguyen** [comment]
ok to test

File: Product/Production/Gateway/DocumentQuery_20/src/main/resources/docquery/_20/webservices.xml [hunk 1]
```diff
@@ -57,6 +60,9 @@ http://cxf.apache.org/core http://cxf.apache.org/schemas/core.xsd ">
         <jaxws:handlers>
             <ref bean="TransactionHandler" />
         </jaxws:handlers>
+        <jaxws:outInterceptors>
```
Comment (jsmith2) [inline Product/Production/Gateway/DocumentQuery_20/src/main/resources/docquery/_20/webservices.xml]: This isn't right.  Needs to only be on the NHIN endpoint.  Need to do that for all 3 services (only one endpoint in each file).

**Comment by jsmith2** [comment]
Need to change the version number for this.  4.5.3-HOTFIX

**Review by jsmith2 — CHANGES_REQUESTED** [review]

File: Product/Production/Gateway/DocumentQuery_30/src/main/resources/docquery/_30/webservices.xml [hunk 2]
```diff
@@ -87,5 +90,8 @@ http://cxf.apache.org/core http://cxf.apache.org/schemas/core.xsd ">
         <jaxws:handlers>
             <ref bean="TransactionHandler" />
         </jaxws:handlers>
+        <jaxws:outInterceptors>
+            <ref bean="HSTSHeaderInterceptor"/>
```
Comment (jsmith2) [inline Product/Production/Gateway/DocumentQuery_30/src/main/resources/docquery/_30/webservices.xml]: This is still wrong.  Don't need it on the entity layer.

**Review by jsmith2 — APPROVED** [review]

**Pull request merged**